### ISSUE TITLE
Accessibility generator for capybara_accessible_selectors fails

### ISSUE THREAD

**Issue by louis-antonopoulos** [thoughtbot/suspenders#1228]
In response to running `rails new`

```
rails new my-app-name \
 --skip_rubocop \
 --skip-test \
 -d=postgresql \
 -m=https://raw.githubusercontent.com/thoughtbot/suspenders/main/lib/install/web.rb
```

the [accessibility generator](https://github.com/thoughtbot/suspenders/blob/1fdbb2287bd156cb1f0a0dd67708c0af680dbdd9/lib/generators/suspenders/accessibility_generator.rb#L15-L20) `#add_capybara_gems` step raises an error:

```
Git error: command `git fetch --force --quiet --no-tags --depth 1 --
https://github.com/citizensadvice/capybara_accessible_selectors.git refs/heads/HEAD:refs/heads/HEAD` in directory
~.asdf/installs/ruby/3.3.3/lib/ruby/gems/3.3.0/cache/bundler/git/capybara_accessible_selectors-7e28584b80a657a0689a0e8e8831f0d4fc91cb20
has failed.
Revision  does not exist in the repository https://github.com/citizensadvice/capybara_accessible_selectors.git. Maybe you
misspelled it?
    generate  suspenders:advisories
       rails  generate suspenders:advisories 
The git source https://github.com/citizensadvice/capybara_accessible_selectors.git is not yet checked out. Please run `bundle install` before trying to start your application
```

According to their [instructions](https://github.com/citizensadvice/capybara_accessible_selectors?tab=readme-ov-file#usage), it should be added to the `Gemfile` with a `tag:` like this:

```
Include in your Gemfile:

group :test do
  # It is recommended you use a tag as the main branch may contain breaking changes
  gem "capybara_accessible_selectors", git: "https://github.com/citizensadvice/capybara_accessible_selectors", tag: "v0.12.0"
end
```

In a new app, including this in the `Gemfile` breaks:

```
gem "capybara_accessible_selectors", git: "https://github.com/citizensadvice/capybara_accessible_selectors"
```

but this succeeds:

```
gem "capybara_accessible_selectors", git: "https://github.com/citizensadvice/capybara_accessible_selectors", tag: "v0.12.0"
```
